### PR TITLE
Cluster PWM channels

### PR DESCRIPTION
**Pull request by anall**
See https://github.com/rp-rs/rp-hal/pull/44#issuecomment-867651257

Will probably make that PR need to do a little less gross macro madness.

### CODE REVIEW

File: svd/rp2040.yaml [hunk 1]
```diff
@@ -40,6 +40,30 @@ PIO0:
       "IRQ*_INTF": {}
       "IRQ*_INTS": {}
 
+PWM:
+  "CH*_TOP":
+    _modify:
+      "CH0_TOP":
+        name: "TOP"
```
Comment (anall) [inline svd/rp2040.yaml:47]: Otherwise you need to do something like `pwm3.top.write(|w| unsafe { w.ch0_top().bits(0xffff) } );` which is… not great

**Review by 9names — COMMENTED** [review]
Looks good but the case change on the fields looks a little suspect - did you intend to make them lower-case?

File: svd/rp2040.yaml [hunk 2]
```diff
@@ -40,6 +40,30 @@ PIO0:
...
+
+  "CH*_CTR":
+    _modify:
+      "CH0_CTR":
+        name: "CTR"
+
+  _cluster:
+    "CH%s":
+      "CH*_CC":
+        name: "cc"
```
Comment (9names) [inline svd/rp2040.yaml:57]: Shouldn't these field names be uppercase?

Comment (anall) [reply svd/rp2040.yaml:57]: I did not intend to make them lower-case. Will fix this evening.

Comment (anall) [reply svd/rp2040.yaml]: and fixed

**Review by 9names — APPROVED** [review]
LGTM

**Pull request merged**